packages/api/templates/index.html

```diff
@@ -554,704 +554,20 @@
-              Destination: 'rhikRdkFw28csKw9z7fVoBjWncz1HSoQij',
-              Amount: '100000',
-              Memos: [
-                {
-                  Memo: {
-                    MemoData: '54657374206D656D6F',
-                    MemoType: '4465736372697074696F6E'
-                  }
-                }
-              ]
-            },
-            {
-              ID: '045',
-              TransactionType: 'Payment',
-              Destination: 'rhikRdkFw28csKw9z7fVoBjWncz1HSoQij',
-              Amount: '100000',
-              Memos: [
-                {
-                  Memo: {
-                    MemoData: '54657374206D656D6F',
-                    MemoType: '4465736372697074696F6E'
-                  }
-                }
-              ]
-            },
-            {
-              ID: '046',
-              TransactionType: 'Payment',
-              Destination: 'rhikRdkFw28csKw9z7fVoBjWncz1HSoQij',
-              Amount: '100000',
-              Memos: [
-                {
-                  Memo: {
-                    MemoData: '54657374206D656D6F',
-                    MemoType: '4465736372697074696F6E'
-                  }
-                }
-              ]
-            },
-            {
-              ID: '047',
-              TransactionType: 'Payment',
-              Destination: 'rhikRdkFw28csKw9z7fVoBjWncz1HSoQij',
-              Amount: '100000',
-              Memos: [
-                {
-                  Memo: {
-                    MemoData: '54657374206D656D6F',
-                    MemoType: '4465736372697074696F6E'
-                  }
-                }
-              ]
-            },
-            {
-              ID: '048',
-              TransactionType: 'Payment',
-              Destination: 'rhikRdkFw28csKw9z7fVoBjWncz1HSoQij',
-              Amount: '100000',
-              Memos: [
-                {
-                  Memo: {
-                    MemoData: '54657374206D656D6F',
-                    MemoType: '4465736372697074696F6E'
-                  }
-                }
-              ]
-            },
-            {
-              ID: '049',
-              TransactionType: 'Payment',
-              Destination: 'rhikRdkFw28csKw9z7fVoBjWncz1HSoQij',
-              Amount: '100000',
-              Memos: [
-                {
-                  Memo: {
-                    MemoData: '54657374206D656D6F',
-                    MemoType: '4465736372697074696F6E'
-                  }
-                }
-              ]
-            },
-            {
-              ID: '050',
-              TransactionType: 'Payment',
-              Destination: 'rhikRdkFw28csKw9z7fVoBjWncz1HSoQij',
-              Amount: '100000',
-              Memos: [
-                {
-                  Memo: {
-                    MemoData: '54657374206D656D6F',
-                    MemoType: '4465736372697074696F6E'
-                  }
+          const transactions = Array.from({ length: 50 }, (_, index) => ({
```

Comment:
You removed the bored apes minting :( (and a setTrustline)

Reply:
Oh I see, we can add this back tomorrow :)